assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+            ]
+        },
+        "HELEN1": {
+            "icao": "HELEN1",
+            "name": "Helen One",
+            "exitPoints": {
+                "SHION": [
+					"WENDY", "SHION"
+				],
+                "SOUJA": [
+					"WENDY", "SHION", "SOUJA"
+				],
+                "WASYU": [
+					"WENDY", "SHION", "WASYU"
+				],
+                "HABAR": [
+					"WENDY", "SHION", "HABAR"
+				],
+                "CHIZU": [
+					"KAWAT", "CHIZU"
+				]
+            },
+            "rwy": {
+                "06L": ["HELEN", "MAIKO"],
+                "06R": ["B6R10", ["B6R11", "A25+"], "HELEN", "MAIKO"],
+                "24L": ["B4L10", ["BB430", "S210"],["HELEN", "A80+"], "MAIKO"],
+                "24R": [["BB430", "S210"], ["HELEN", "A80+"], "MAIKO"]
+            },
+            "draw": [
+                ["HELEN", "MAIKO"],
+                ["B6R11", "HELEN"],
+                ["BB430", "HELEN"],
+                ["MAIKO", "WENDY", "SHION*"],
+                ["SHION", "SOUJA*"],
+                ["SHION", "WASYU*"],
+                ["SHION", "HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "IWAYA1": {
+            "icao": "IWAYA1",
+            "name": "Iwaya One",
+            "exitPoints": {
+                "SHION": [
+					"WENDY", "SHION"
+				],
+                "SOUJA": [
+					"WENDY", "SHION", "SOUJA"
+				],
+                "WASYU": [
+					"WENDY", "SHION", "WASYU"
+				],
+                "HABAR": [
+					"WENDY", "SHION", "HABAR"
+				],
+                "CHIZU": [
+					"KAWAT", "CHIZU"
+				]
+            },
+            "rwy": {
+                "24L": ["BB430", ["HELEN", "A80+"], "MAIKO"],
+                "24R": ["BB430", ["HELEN", "A80+"], "MAIKO"]
+            },
+            "draw": [
+                ["HELEN"],
+                ["SHION*"],
+                ["SOUJA*"],
+                ["WASYU*"],
+                ["HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "DAISY1": {
+            "icao": "DAISY1",
+            "name": "Daisy One",
+            "exitPoints": {
+                "HIBIS": [
+					"HIBIS"
+				]
+            },
+            "rwy": {
+                "06L": ["DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R12", "DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24L": ["B4L10", ["BB430", "S210"], ["DAISY", "A60"], ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24R": [["DAISY", "A60+"], ["JULIA", "A80+"], "LINDA", "GLORY"]
+            },
+            "draw": [
+                ["DAISY", "JULIA", "LINDA", "GLORY"],
+                ["B6R11", "B6R12", "DAISY"],
+                ["BB430", "DAISY"],
+                ["GLORY", "HIBIS*"]
+            ]
+        }
+    },
+    "stars": {
+        "AKASI": {
+            "icao": "AKASI",
+            "name": "Akasi",
+            "entryPoints": {
+                "TOKUSHIMA": [
+					"TSC"
+				]
```

Comment:
Due to the need to type the name of the entry point when assigning aircraft a new STAR, please use the proper ICAO fix/navaid identifier for all entry and exit points:
```
"entryPoints": {
    "TSC": [
        "TSC"
    ]
}
```